### PR TITLE
Fix career page performance

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
Solve lighthouse Errors to improve performance on **CAREER** and **PORTFOLIO PAGE** in mobile.
CAREER PAGE:
Previous: 89
**Now: 91-92 approx.**

PORTFOLIO PAGE:
Previous: 85
**Now: 98**

- [x] Links do not have a discernible name.
- [x] Does not use passive listeners to improve scrolling performance
- [x] Eliminate render-blocking resources.
- [x] Background and foreground colors do not have a sufficient contrast ratio.
- [x] Links do not have descriptive text

### CODE REVIEW

File: vue-frontend/package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "vue-router": "^4.0.14"
   },
   "dependencies": {
+    "@fullhuman/postcss-purgecss": "^5.0.0",
```
Comment (cp-sumi-k) [inline vue-frontend/package.json:55]: I think you need `purgecss`, its `postcss-purgecss`

File: vue-frontend/package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "@fortawesome/free-regular-svg-icons": "^6.1.1",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/vue-fontawesome": "^3.0.0-5",
+    "@fullhuman/postcss-purgecss": "^5.0.0",
```
Comment (cp-sumi-k) [inline vue-frontend/package.json:21]: Remove `purgecss` and `cssnano`, use `purge` instead in `tailwind.config`

File: vue-frontend/src/components/home-new/HomeIndex.vue [hunk 3]
```diff
@@ -62,6 +62,7 @@ export default {
   },
   mounted() {
     this.$gtag.event("view_landing_section");
+    document.addEventListener("touchstart", handler, { passive: true });
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/HomeIndex.vue:65]: What is purpose of it?

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 4]
```diff
@@ -85,6 +85,7 @@
                   @click.native="$gtag.event('tap_read_more_job')"
                   class="gradient-border-btn tw-p-[11px] sm:tw-p-[16px]"
                   :to="'/jobs/' + career.unique_id"
+                  aria-label="read more about the designation and work"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:88]: Keep it as `read more`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/HomeIndex.vue (hunk 3 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/HomeIndex.vue:65]: To solve this "Does not use passive listeners to improve scrolling performance" . I found this only solution for it.

Comment (cp-sumi-k) [reply vue-frontend/src/components/home-new/HomeIndex.vue:65]: Shouldn't we need to pass our defined handler in `handler`?

Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/HomeIndex.vue:65]: Passive listeners do not work on scroll events.

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 5]
```diff
@@ -18,7 +18,6 @@
             :srcset="`${portfolio.images[0]} 400w, ${portfolio.images[1]} 800w, ${portfolio.images[2]} 1200w, ${portfolio.images[3]} 1600w`"
             sizes="(min-width: 992px) 45vw, 100vw"
             class="tw-w-full tw-h-full tw-object-cover"
-            loading="lazy"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:21]: I think it should not be removed

File: vue-frontend/tailwind.config.cjs [hunk 6]
```diff
@@ -2,7 +2,9 @@ const plugin = require("tailwindcss/plugin");
 const defaultTheme = require("tailwindcss/defaultTheme");
 module.exports = {
   prefix: "tw-",
+  purge: ["./index.html", "./src/**/*.{vue,js,ts,jsx,tsx}"],
```
Comment (cp-sumi-k) [inline vue-frontend/tailwind.config.cjs:5]: I think purge is replaced by content, So use content

**Review by cp-sumi-k — APPROVED** [review]

File: vue-frontend/src/components/portfolio/DesignSection.vue [hunk 7]
```diff
@@ -95,6 +95,7 @@
 import AspectRatio from "@/components/utils/AspectRatio.vue";
 import LottieAnimation from "@/components/utils/LottieAnimation.vue";
 import { analyticsEvent } from "@/utils.js";
+import { faArrowTrendUp } from "@fortawesome/free-solid-svg-icons";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/DesignSection.vue:98]: What is use of it?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/VideoSection.vue [hunk 8]
```diff
@@ -260,7 +260,7 @@ export default {
     if (window.innerWidth <= 992) {
       this.isShowSliderInMobile = true;
     }
-    window.addEventListener("scroll", this.sendEvent);
+    window.addEventListener("scroll", this.sendEvent, { passive: true });
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/VideoSection.vue:263]: I think passive is only for `touch` or `wheel` events

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/jobs/VirtuesView.vue [hunk 9]
```diff
@@ -101,7 +101,7 @@
           class="tw-relative tw-flex-[100%] md:tw-flex-[33%]"
           v-for="virtue in virtues.slice(5, 8)"
           :key="virtue.id"
-          @touchstart="activeIndex = virtue.id"
+          @touchstart.passive="activeIndex = virtue.id"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/VirtuesView.vue:104]: Have you checked it with real mobile devices?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/BlogSection.vue [hunk 10]
```diff
@@ -49,6 +49,7 @@
               :src="blog.thumbnail"
               class="tw-w-full tw-h-full tw-object-cover tw-rounded-t-x-lg"
               loading="lazy"
+              alt="blog-background"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/BlogSection.vue:52]: Give the appropriate alt text according the blog post

File: vue-frontend/src/components/home-new/ContributeSection.vue [hunk 11]
```diff
@@ -43,6 +43,7 @@
           :src="contribution.image"
           class="tw-w-full tw-h-full tw-object-cover tw-rounded-[10px]"
           loading="lazy"
+          alt="contribution-image"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/ContributeSection.vue:46]: Give the appropriate alt text according the library

File: vue-frontend/src/components/jobs/VirtuesView.vue (hunk 9 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/jobs/VirtuesView.vue:104]: yes, its working

**Review by cp-shruti-s — APPROVED** [review]

File: vue-frontend/package-lock.json [hunk 12]
```diff
@@ -2885,7 +2885,6 @@
         "copy-webpack-plugin": "^9.0.1",
         "css-loader": "^6.5.0",
         "css-minimizer-webpack-plugin": "^3.0.2",
-        "cssnano": "^5.0.0",
```
Comment (cp-sumi-k) [inline vue-frontend/package-lock.json:2888]: We can not directly remove dependency from the package-lock. Need to uninstall package with  `npm uninstall cssnano` command

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/package-lock.json (hunk 12 above)
Comment (cp-priyanka-g) [reply vue-frontend/package-lock.json:2888]: i have used that command only.

**Pull request closed without merging**